### PR TITLE
added seed file for database

### PR DESCRIPTION
**Pull request by ryan280**

### CODE REVIEW

File: server/tasks/seed.ts [hunk 1]
```diff
@@ -0,0 +1,423 @@
+// FYI when i did /config/mongoConnection absolute pathing it errored so ya'll can fix that
```
Comment (roccopolimen) [inline server/tasks/seed.ts]: That's fine I only set up the absolute pathing for the clientside code

**Comment by roccopolimen** [comment]
Made changes to add typings

@ryan280 you still need to add firebaseId to the user
also for the location field, can you populate with actual phone numbers or links for the locations instead of "Phone number" or "Online" so it looks a bit more like we'll actually be using it.

**Comment by mpolimen** [comment]
Other than Rocco's remarks, LGTM 👍

**Comment by roccopolimen** [comment]
@ryan280 waiting for fixes...

**Review by roccopolimen — APPROVED** [review]
LGTM

**Comment by ryan280** [comment]
I kept online because it feels unnecessary to have links to the application, when that information is already stored in the application document

**Pull request merged**